src/api/java/CMakeLists.txt

```diff
@@ -2,28 +2,112 @@
 ## CMakeLists.txt
 ## Top contributors (to current version):
 ##   Mudathir Mohamed
-## This file is part of the CVC4 project.
+## This file is part of the CVC5 project.
 ## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
 ## in the top-level source directory and their institutional affiliations.
 ## All rights reserved.  See the file COPYING in the top-level source
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# create directories
+file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/main/java/cvc5/")
+file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/jni")
```

Comment:
This directory is for the generated JNI files, right? I think that it would be more consistent to have them in `src/api/java/jni`.

Reply:
Moved jni files to `src/api/java/jni`